shell-darwin.nix

```diff
@@ -0,0 +1,30 @@
+# @zupo uses this file to work on nix.dev on his M1 Monterey
```

Comment:
Note that this isn't tested on the CI, so it will likely break with updates

Reply:
Yeah, that's fine, it's mostly for me.